holoviews/operation/datashader.py

```diff
@@ -264,6 +264,10 @@ class shade(ElementOperation):
         and any valid transfer function that accepts data, mask, nbins
         arguments.""")
 
+    link_inputs = param.Boolean(default=True, doc="""
+         Whether a plot created from the output of this operation
+         should be linked to the inputs of this operation.""")
```

Comment:
Docstring should now be about what this means for the ``shade`` operation. Something like:

'By default, the link_inputs parameter is set to True so that when applying shade ...'